hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -45,6 +45,8 @@
     ALLOWED_HOSTS = ["newhacks.ca", "www.newhacks.ca"]
     CORS_ORIGIN_REGEX_WHITELIST = [
         r"^https://(?:www\.)?newhacks.ca:?\d*$",
+        r"^https://(?:www\.)?newhacks.ca(/\w+)+"
+        r"^http://(?:www\.)?hardware.newhacks.ca(/\w+)+",
```

Comment:
We can probably simplify this to:

```suggestion
        r"^https://(?:www\.)?newhacks.ca(/\w+)+"
        r"^https://hardware.newhacks.ca(/\w+)+",
```

No need to allow ports on the main domain, and want to use https for the hardware site